Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby:2.2.3
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
Ah, I did just notice `nodejs` in here. Is this something your fork requires, perhaps?